stable/prometheus-operator/requirements.lock

```diff
@@ -4,9 +4,9 @@ dependencies:
   version: 2.3.1
 - name: prometheus-node-exporter
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 1.5.2
+  version: 1.7.0
```

Comment:
Since the requirement bumped the minor version, I think we should bump our minor version too (as opposed to the patch version)

Reply:
Yep, changed chart version to 6.17.0